extensions/ql-vscode/src/pure/result-keys.ts

```diff
@@ -37,7 +52,7 @@ export function getResult(sarif: sarif.Log, key: Result): sarif.Result | undefin
 /**
  * Looks up a specific path in a result set.
  */
-export function getPath(sarif: sarif.Log, key: Path): sarif.ThreadFlow | undefined {
+export function getPath(sarif: sarif.Log, key: Path | PathNode): sarif.ThreadFlow | undefined {
```

Comment:
Unused.

Reply:
The type is needed from both call sites to `getPath` - this is due to the change in the `Path` type.

Reply:
Sorry...missed that the first time.